docs/usage/configuration-options.md

```diff
@@ -2664,6 +2664,45 @@ This will lead to following update where `1.21-alpine` is the newest version of
 image: my.new.registry/aRepository/andImage:1.21-alpine
 ```
 
+E.g.: Moving where gitlab-ci files are stored on a server and updating `includes:` definitions
+
+You may find you need to specify a second matchString for the new name, as when the regex manager validates the change it will not always detect that the new value is valid
```

Comment:
```suggestion
You may find you need to specify a second `matchString` for the new name, as when the regex manager validates the change it will not always detect that the new value is valid.
```